pkg/platform/baremetal/template.go

```diff
@@ -47,6 +47,14 @@ module "bare-metal-{{.ClusterName}}" {
   worker_names       = {{.WorkerNames}}
   worker_macs        = {{.WorkerMacs}}
   worker_domains     = {{.WorkerDomains}}
+
+  {{- if .Labels}}
+  labels = [
+  {{- range .Labels}}
+    "{{.}}",
```

Comment:
Can you please assign a name instead? This seems quite cryptic without any reason, IMHO.

Reply:
OK